<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ===== Style général ===== */
QDialog {
    background-color: #2d2d30;  /* Fond sobre et moderne */
    border: 1px solid #3e3e40; /* Bordure fine et discrète */
    border-radius: 10px;       /* Coins arrondis élégants */
    padding: 15px;
    color: #e0e0e0;            /* Texte gris clair pour une bonne lisibilité */
}

/* ===== QLabel ===== */
QLabel {
    font-size: 13px;
    font-weight: bold;         /* Texte légèrement mis en valeur */
    color: #dcdcdc;
    margin: 5px 0;
}

/* ===== QLineEdit ===== */
QLineEdit {
    background-color: #39393d; /* Fond sombre et uni */
    border: 1px solid #5a5a5d; /* Bordure subtile */
    border-radius: 5px;        /* Coins légèrement arrondis */
    color: #e0e0e0;            /* Texte clair */
    padding: 5px;
    font-size: 14px;
}
QLineEdit:focus {
    border: 1px solid #6fa3ef; /* Bordure bleue moderne au focus */
    background-color: #3f3f43; /* Fond légèrement plus clair */
}

/* ===== QPushButton ===== */
QPushButton {
    background-color: #454547; /* Bouton sobre et uniforme */
    color: #e0e0e0;            /* Texte clair */
    font-size: 14px;
    font-weight: bold;
    border: 1px solid #5a5a5d; /* Bordure subtile */
    border-radius: 5px;        /* Coins arrondis */
    padding: 7px 15px;
}
QPushButton:hover {
    background-color: #565659; /* Fond plus clair au survol */
}
QPushButton:pressed {
    background-color: #666669; /* Fond encore plus clair au clic */
}
QPushButton:disabled {
    background-color: #3c3c3f; /* Fond désaturé pour boutons désactivés */
    color: #8a8a8c;
}

/* Bouton spécifique pour OK */
QPushButton#pushButtonOk {
    background-color: #3a875e; /* Vert moderne */
    border: 1px solid #469e6d;
}
QPushButton#pushButtonOk:hover {
    background-color: #41a173; /* Vert plus lumineux au survol */
}
QPushButton#pushButtonOk:pressed {
    background-color: #369462; /* Vert plus sombre au clic */
}

/* Bouton spécifique pour Cancel */
QPushButton#pushButtonCancel {
    background-color: #8b3b3b; /* Rouge doux */
    border: 1px solid #a04c4c;
}
QPushButton#pushButtonCancel:hover {
    background-color: #a04c4c; /* Rouge plus lumineux au survol */
}
QPushButton#pushButtonCancel:pressed {
    background-color: #7b3535; /* Rouge plus sombre au clic */
}

QLabel#labelWarningPassword {
    color: #8b3b3b; /* Rouge doux */
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEditPassword_2">
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>Connection</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditLastName">
     <property name="placeholderText">
      <string>First name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelWarningPassword">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Both passwords don't match</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditFistName">
     <property name="placeholderText">
      <string>First name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>At list 6 caracters </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="listWidgetAvatar"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditFistName</tabstop>
  <tabstop>lineEditLastName</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditPassword_2</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SignUpDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
